x-pack/plugins/ingest_manager/server/saved_objects/index.ts

```diff
@@ -202,21 +202,23 @@ const savedObjectTypes: { [key: string]: SavedObjectsType } = {
         inputs: {
           type: 'nested',
           properties: {
-            type: { type: 'keyword' },
-            enabled: { type: 'boolean' },
-            processors: { type: 'keyword' },
-            config: { type: 'flattened' },
-            vars: { type: 'flattened' },
+            type: { type: 'keyword', index: false },
+            enabled: { type: 'boolean', index: false },
+            processors: { type: 'keyword', index: false },
+            config: { type: 'flattened', index: false },
+            vars: { type: 'flattened', index: false },
             streams: {
               type: 'nested',
               properties: {
-                id: { type: 'keyword' },
-                enabled: { type: 'boolean' },
-                dataset: { type: 'keyword' },
-                processors: { type: 'keyword' },
-                config: { type: 'flattened' },
-                agent_stream: { type: 'flattened' },
-                vars: { type: 'flattened' },
+                id: { type: 'keyword', index: false },
+                enabled: { type: 'boolean', index: false },
+                dataset: { type: 'keyword', index: false },
+                processors: { type: 'keyword', index: false },
+                config: { type: 'flattened', index: false },
+                // @ts-ignore doc_values is not in SavedObjectsFieldMapping
+                agent_stream: { type: 'flattened', index: false, doc_values: false },
```

Comment:
@elastic/kibana-platform Do you think I should update the type `SavedObjectsFieldMapping` to add definition for `doc_values` ?